org-configurations/default-configuration.yml

```diff
@@ -0,0 +1,88 @@
+---
+name: "default name"
+slug: "default-slug"
+
+# configuration variables for the server app
+server:
```

Comment:
@nemesisdesign I wrote this default config. I have separated variables in server and client section. Variables in the server section will only be available to the server and will prevent sensitive info like the secret key to be exposed to a client. The urls are highly customizable and organizations can set different host servers for proxy and also customize the url patterns for different views.  What do you think about it?

Reply:
very good!